Sources/BaseButtonBarPagerTabStripViewController.swift

```diff
@@ -0,0 +1,294 @@
+/*****************************************************************************
+ * BaseButtonBarPageTabStripViewController.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <caro # videolan.org>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+
+class VLCLabelCell: UICollectionViewCell {
+    static var cellIdentifier = "VLCLabelCell"
```

Comment:
I think a `static let` might be better for an identifier.

Reply:
very true I think I have this somewhere else as well so I will change that